frontend/src/components/RisLawPreview.vue

```diff
@@ -24,18 +26,177 @@ withDefaults(
      * @default false
      */
     highlightAffectedDocument?: boolean
+
+    /**
+     * EIds of the currently selected elements.
+     */
+    selected?: string[]
   }>(),
   {
     highlightMods: false,
     highlightAffectedDocument: false,
+    selected: () => [],
+  },
+)
+
+const emit = defineEmits<{
+  /**
+   * Event fired when clicking on an HTML-element for the specified LDML.de element.
+   *
+   * Event handlers, a11y properties and so on are only created for elements for which an event handler is registered.
+   *
+   * E.g. `@click:akn:article` will fire for every click on a part of the preview that is showing an article.
+   */
+  [key: `click:akn:${string}`]: [
```

Comment:
Question: Does this work with TypeScript, Intellisense etc.? We should do it like this either way but it would be really cool if the tooling was smart enough to pick this up

Reply:
sadly it doesn't work with most of those things :/